apps/emqx_authn/src/simple_authn/emqx_authn_redis.erl

```diff
@@ -149,8 +149,8 @@ authenticate(
                     of
                         ok ->
                             {ok, emqx_authn_utils:is_superuser(Selected)};
-                        {error, Reason} ->
-                            {error, Reason}
+                        {error, _Reason} ->
+                            ignore
```

Comment:
How does this differ from password error and not found ?

Reply:
I think the root cause of this problem is that redis returns the empty list as `[[nil, nil, nil]]` instead of  `[]` 
see: https://github.com/emqx/emqx/issues/8800#issuecomment-1226797455